drivers/hud_apr/app/controllers/hud_apr/questions_controller.rb

```diff
@@ -6,6 +6,7 @@
 
 module HudApr
   class QuestionsController < BaseController
+    include HudApr::CellDetailsConcern
```

Comment:
I think this needs to come out (could be wrong).

Reply:
✅ Yes. No need for it here.